smh_app/utils.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+from smh_app.settings import SOCIAL_AUTH_VMI_HOST
+
+
+def get_vmi_user_data(request):
+    """ Makes a call to the VMI user_profile endpoint and returns a response """
+    user_endpoint = SOCIAL_AUTH_VMI_HOST + '/api/v1/user/'
+    social_auth_extra_data = request.user.social_auth.values().first()['extra_data']
+    token = social_auth_extra_data['access_token'] if social_auth_extra_data else None
+    response = requests.get(
```

Comment:
This endpoint now has a filter that should let you include search params so it can do the heavy lifting.

https://github.com/TransparentHealth/vmi/blob/master/apps/api/views/user.py#L27-L38